lib/migrations/20190530163257_track_typeOfTest.js

```diff
@@ -0,0 +1,31 @@
+const _ = require('underscore');
+
+exports.up = async function(knex, Promise) {
+    await knex.schema.table('students', function(t){
+        t.string('typeOfTest').after('stage').nullable();
+    });
+
+    let students = await knex('students').select('*');
+    let promises = [];
+    
+    _.each(students, (student) => {
+        if (student.partnerId) {
+            promise = knex('students').update({
```

Comment:
You need to specify the for which student are you updating it, or all the student would be updated every time.

Reply:
you means comment or anything else?

Reply:
studentId for which student are your updating it?

Reply:
it is done